### PR TITLE
feature: Display high/low estimate or just low, not high

### PR DESCRIPTION
**Pull request by bhoggard**
Completes https://artsyproduct.atlassian.net/browse/CX-910.

Screen Shot 2020-12-17 at 3 10 36 PM

### CODE REVIEW

File: app/views/admin/offers/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <div class='page-title'>
-  <%= link_to 'Create New', new_admin_offer_path, class: 'btn btn-small btn-primary pull-right' %>
```
Comment (jonallured) [inline app/views/admin/offers/index.html.erb:2]: Question: I'm unclear if this was intentional - can you confirm removing this link was meant to be a part of this PR? 🤔

File: app/models/offer.rb [hunk 2]
```diff
@@ -61,6 +61,14 @@ def set_state
...
   end
 
+  def estimate_display
+    if high_estimate_cents.present? && low_estimate_cents.present?
+      "#{low_estimate_display} - #{high_estimate_display}"
+    else
+      low_estimate_display
+    end
+  end
+
```
Comment (jonallured) [inline app/models/offer.rb:71]: I'd like to see a unit test for this if you don't mind!

**Review by jonallured — COMMENTED** [review]
Nice, just a couple comments for you. ❤️

File: app/views/admin/offers/index.html.erb (hunk 1 above)
Comment (bhoggard) [reply app/views/admin/offers/index.html.erb:2]: Yes, I realized while working on this that we don't create offers this way in the controller. It's done from a submission page.

File: app/models/offer.rb (hunk 2 above)
Comment (bhoggard) [reply app/models/offer.rb:71]: next PR!

Comment (bhoggard) [reply app/models/offer.rb:71]: There is already a helper. I'll replace this in the next PR.

File: app/views/admin/offers/index.html.erb (hunk 1 above)
Comment (bhoggard) [reply app/views/admin/offers/index.html.erb:2]: (It was copied over from another form incorrectly).

File: app/helpers/offers_helper.rb [hunk 3]
```diff
@@ -66,10 +66,14 @@ def offer_type_description(offer)
...
 
   def estimate_display(offer)
+    return unless offer
+
     currency = Money::Currency.new(offer.currency)
     estimate =
       [offer.low_estimate_cents, offer.high_estimate_cents].compact
-        .map { |amt| (amt / currency.subunit_to_unit).round }.join(' - ')
+        .map do |amt|
+        number_with_delimiter(amt / currency.subunit_to_unit.round)
```
Comment (bhoggard) [inline app/helpers/offers_helper.rb:75]: This didn't show commas for big numbers, it does now.

Comment (bhoggard) [reply app/helpers/offers_helper.rb:75]: Screen Shot 2020-12-17 at 5 22 36 PM

**Pull request merged**